spec/index.bs

```diff
@@ -5245,7 +5245,7 @@ dictionary GPUVertexAttribute {
                 Otherwise:
                 - |attrib|.{{GPUVertexAttribute/offset}} + sizeof(|attrib|.{{GPUVertexAttribute/format}}) &le;
                     |descriptor|.{{GPUVertexBufferLayout/arrayStride}}.
-            - |attrib|.{{GPUVertexAttribute/offset}} is a multiple of the size of one component of
+            - |attrib|.{{GPUVertexAttribute/offset}} is a multiple of the minimum of 4 and the size of
```

Comment:
this would be a problem if we wanted to add formats, which sizes are not divisible by 4. E.g. `Snorm8x3`. I suppose the extension can also write the prose to override this alignment requirement.

Reply:
Yeah. I assume these wouldn't be available in D3D12 though. Not sure how it would be specced. Maybe a per-format hardcoded alignment in the spec? Or a flexible alignment extension?

Reply:
We could just update it to match the D3D11 spec:

> The alignment of an Element in a structure in memory must be, in bytes, the nearest power of 2 greater or equal to the width of the Element's format, or 4, whichever is less. Thus alignment is always at 1 byte, 2 bytes or 4 byte granularity. This alignment is measured from the starting address of the structure. The starting address of the structure must also maintain alignment of all its members.

https://crbug.com/dawn/1010#c3